packages/typespec-ts/test/integration/generated/azure/resource-manager/models/resources/samples-dev/nestedProxyResourcesCreateOrReplaceSample.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import createAzureArmResourceClient, {
+  getLongRunningPoller,
+} from "@azure/arm-resources";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to call operation CreateOrReplace
+ *
+ * @summary call operation CreateOrReplace
+ */
+async function nestedProxyResourcesCreateOrReplaceSample() {
+  const client = createAzureArmResourceClient();
```

Comment:
Shall we exclude RLC samples for resource manager SDKs?

Reply:
seems we are generating RLC for mgmt. SDK,  is this needed?

Reply:
We don't object to generate standalone RLC for mgmt plane and if we split RLC from Modular, both will be a requirement for mgmt plane, though modular is the current recommendation here. Another consideration is this is for test coverage.